src/casp/services/api/services/authorization.py

```diff
@@ -0,0 +1,34 @@
+import typing as t
+
+from casp.services.api.data_manager import CellariumGeneralDataManager
+from casp.services.api.data_manager import exceptions as dm_exc
+from casp.services.api.services import exceptions
+from casp.services.db import models
+
+
+class Authorizer:
```

Comment:
Why make this a separate class? It has only one method.

Reply:
I don't know if that's a reason not to have it in there.  Logically it seems to make more sense in here than in the cell operations service so I wanted to pull it out. I originally pulled it out because I thought we were doing a similar thing in the general ops service and while we're not currently, it seems like we might and it makes sense (IMO) to have a place for authz checks like this to live.